source/appModules/winword.py

```diff
@@ -54,7 +54,10 @@ class WinwordWordDocument(WordDocument):
 	def _get_description(self) -> str:
 		try:
 			curView = self.WinwordWindowObject.view.Type
-			return f"{ViewType(curView).displayString} {super()._get_description()}"
+			description = super().description
+			if isinstance(super()._get_description(), str):
+				return f"{ViewType(curView).displayString} {description}".strip()
+			return curView.displayString
 		except AttributeError:
 			return super()._get_description()
```

Comment:
```suggestion
			return description
```

Reply:
@LeonarddeR - is this resolved?

Reply:
I think we shouldn't apply this suggestion since the description variable is created in try, not in Except,